arm9/source/game/codelzss.c

```diff
@@ -91,3 +97,276 @@ u32 DecompressCodeLzss(u8* code, u32* code_size, u32 max_size) {
     *code_size = data_end - data_start;
     return 0;
 }
+
```

Comment:
Starting here: That code could use a cleanup. More consistency in function names (use lower case for all 'private' functions for example), and no newlines before `{`. Part of this is pretty hard to read, too. I'm also wondering why that has to take that long. I do understand that LZSS is an asymmetric compression algo, but the requirement here seems a bit harsh.